lib/Alchemy/Phrasea/Core/Event/DataboxEvent/DataboxEvent.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/*
+ * This file is part of Phraseanet
+ *
+ * (c) 2005-2014 Alchemy
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Alchemy\Phrasea\Core\Event\DataboxEvent;
+
+use Symfony\Component\EventDispatcher\Event as SfEvent;
+
+class DataboxEvent extends SfEvent
```

Comment:
No need for `as SfEvent`, just extend `Event` no?
This should also be an abstract class.